test/models/report_test.rb

```diff
@@ -3,7 +3,18 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'user can not edit report' do
+    report = reports(:first)
+    assert_not report.editable?(users(:bob))
+  end
+
+  test 'user can edit report' do
+    report = reports(:first)
+    assert report.editable?(users(:alice))
+  end
+
+  test 'Get the date when the daily report was created' do
+    report = reports(:second)
+    assert_equal Date.parse('Tue, 11 Jun 2024'), report.created_on
```

Comment:
```suggestion
    assert_equal "2024-06-11".to_date, report.created_on
```

の方が読みやすいかも？

特に曜日が正しいかどうかは確認するのが大変なので省略した方がいいです